app/javascript/mastodon/actions/store.js

```diff
@@ -5,8 +5,7 @@ export const STORE_HYDRATE_LAZY = 'STORE_HYDRATE_LAZY';
 
 const convertState = rawState =>
   fromJS(rawState, (k, v) =>
-    Iterable.isIndexed(v) ? v.toList() : v.toMap().mapKeys(x =>
-      Number.isNaN(x * 1) ? x : x * 1));
+    Iterable.isIndexed(v) ? v.toList() : v.toMap());
```

Comment:
This is just default fromJS(rawState) behaviour, so custom function could be removed

    const convertState = rawState => fromJS(rawState)

Reply:
Unfortunately this is from a commit in #5019, so I'm not sure how much sense it would make to revise it here. I can do that, or you can just make that change in master. (Or I could submit another PR for it, if you really want.)